binder/environment.yml

```diff
@@ -5,5 +5,6 @@ channels:
 dependencies:
   - python=3
   - cartopy
+  - ffmpeg
   - openblas
```

Comment:
Do you know why are you depending on openblas?
Maybe we should remove this in the OceanParcels fork

Reply:
This is to enforce numpy et al from the conda-forge channel.  Otherwise there may be cases where numpy from `defaults` is considered newer.  This sometimes mixes up envs.